src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -59,24 +76,27 @@ function Caption({ icon: Icon = AlertTriangle, caption, color = 'secondary', too
   )
 }
 
-function GasEstimate({ gasUseEstimateUSD }: GasEstimateProps) {
+function GasEstimate({ gasUseEstimateUSD, trade }: GasEstimateProps) {
   const isWideWidget = useIsWideWidget()
+  if (gasUseEstimateUSD == null) {
```

Comment:
`===`.
Also, this is a guard statement, so feel free to one-line it: `if (!gasUseEstimateUSD) return null`.